### ISSUE TITLE
[DOC] Groupby.std() is a currently non existent function that needs to be removed from Docs

### ISSUE THREAD

**Issue by taureandyernv** [rapidsai/docs#15]
verified by Dante, Paul and others, Groupby.std() does not exist.  We should remove that from our official docs to avoid confusion and distrust of our docs

Location: https://rapidsai.github.io/projects/cudf/en/0.6.0/api.html#cudf.groupby.legacy_groupby.Groupby.std

**Comment by raydouglass** [comment]
@dillon-cullinan Can you look into this? These docs should have been automatically generated right? So how was a non-existant function included?

**Comment by dantegd** [comment]
As the name implies, `legacy_groupby` was the older group by code and thats why the function is there, I believe it might be totally stale but it might still be used for numba function execution, @kkraus14 or @thomcom  would probably know

**Comment by thomcom** [comment]
Yeah `legacy_groupby` auto generated those docs from the set of `_NAMED_FUNCTIONS` that are named in the class - however std, var, and sum_of_squares were never implemented. Tests that use them are automatically skipped. The easiest solution is to remove them from `_NAMED_FUNCTIONS` in `legacy_groupby.py`. For 0.8 probably.
